docs/documentation/api.md

```diff
@@ -313,7 +313,7 @@ expected output
 ### Create new redirect
 
     curl -X POST \
-    -d '{"action":"302", "condition": "/path", "destination":"/new-path", "preserve_query_params": false, "priority": 1, "assertions": [{"expected_location": "/new-path", "given_url": "http://mydomain.example/newpath"}]}' \
+    -d '{"action":"302", "condition": "req.method == \"GET\" && req.url ~ \"/path\"", "src_domain": "example.com", "destination":"/new-path", "preserve_query_params": false, "required_custom_header": false, "priority": 1, "assertions": [{"expected_location": "/new-path", "given_url": "http://mydomain.example/newpath"}]}' \
```

Comment:
If **src_domain** is present **given_url** should contain same domain. Because this redirect is within same domain.
I know this is an example but I prefer to have working examples.

Reply:
fixed